cns/client_test.go

```diff
@@ -1251,14 +1260,98 @@ func TestClient(t *testing.T) {
 		}
 		t.Logf("volume:%q deleted sucessfully", volumeID)
 	}
+
+	// Test CnsReconfigVolumePolicy API
+	if spbmPolicyId4Reconfig != "" {
+		var cnsVolumeCreateWithProfileSpecList []cnstypes.CnsVolumeCreateSpec
+		cnsVolumeCreateSpecWithProfile := cnstypes.CnsVolumeCreateSpec{
```

Comment:
cnsVolumeCreateSpecWithProfile -> There is no profile in this spec.

Reply:
my bad, will fix the names